charts/opensearch-cluster/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: opensearch-cluster
+description: A Helm chart for OpenSearch Cluster
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
```

Comment:
Please remove these comments also. They are a bit misleading as the automatic release process takes care of setting/updating the versions.